src/rpc/mining.cpp

```diff
@@ -861,7 +861,10 @@ static RPCHelpMan getblocktemplate()
 
     UniValue aRules(UniValue::VARR);
     aRules.push_back("csv");
-    if (!fPreSegWit) aRules.push_back("!segwit");
+    if (!fPreSegWit) {
+        aRules.push_back("!segwit");
+        aRules.push_back("!taproot");
```

Comment:
@luke-jr like this?

> This will need `aRules` in `rpc/mining.cpp` updated

Reply:
Oh wait, without `!`

I'll push a fix and add a test...

Before and after this change it should be:

```
bitcoin-cli getblocktemplate '{"rules": ["segwit"]}' | jq '.rules'
[
  "csv",
  "!segwit",
  "taproot"
]
```